docs/arcadia-users-group/20230228-intro-to-python-1/lesson.ipynb

```diff
@@ -0,0 +1,884 @@
+{
```

Comment:
<p>I think it's hard to show the value of functions here with just hello_world. So maybe in a later section, I'd do a more complicated function that allows you to abstract away some complexity. Maybe it could be a point a to point b distance calculation or something. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/arcadia-computational-training/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
I think some of that hopefully will come through in the problem set!